code/modules/admin/topic.dm

```diff
@@ -1760,7 +1760,7 @@
 		P.faxmachineid = fax.UID()
 		P.loc = fax.loc // Do not use fax.receivefax(P) here, as it won't preserve the type. Physically teleporting the fax paper is required.
 		if(istype(H) && H.stat == CONSCIOUS && (istype(H.l_ear, /obj/item/radio/headset) || istype(H.r_ear, /obj/item/radio/headset)))
-			to_chat(H, "Your headset pings, notifying you that a reply to your fax has arrived.")
+			to_chat(H, "<span class='warning'>Your headset pings, notifying you that a reply to your fax has arrived.</span>")

```

Comment:
I'd suggest a class other than 'warning'. 'Warning' messages might be small red font, but they're also some of the most common chat messages. If you want it to stand out, you need to use purple or some other rarely-used color. Also, 'warning' messages are in italics, if memory serves, which makes them harder to read.

Reply:
Ok I changed it to the danger class

Reply:
Include a screenshot of how that looks?

Reply:
It's the same class when someone attacks you

Reply:
I thought that was 'userdanger'.

Reply:
Hm let me check

Reply:
Yeah I changed it to userdanger